test/e2e/tests/ppom/ppom-blockaid-alert-networks-support.spec.js

```diff
@@ -48,7 +50,7 @@ async function mockInfuraWithMaliciousResponses(mockServer) {
 
 describe('PPOM Blockaid Alert - Multiple Networks Support @no-mmi', function () {
   // eslint-disable-next-line mocha/no-skipped-tests
-  it.skip('should show banner alert after switchinig to another supported network', async function () {
+  it.skip('should show banner alert after switching to another supported network', async function () {
```

Comment:
Maybe reenable this test?

Reply:
Done [here](https://github.com/MetaMask/metamask-extension/pull/28565/commits/5d6baf06d1a1d1ecf2a6ee4a5e03f92743fa4767).